src/main/java/updater/UpdateManager.java

```diff
@@ -0,0 +1,159 @@
+package updater;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import ui.UpdateProgressWindow;
+
+import java.io.File;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+/**
+ * UpdateManager is the class that will handle updating of HubTurbo application
```

Comment:
Minimize mentioning class/method names in comments. The more you mention names, the higher the chance of comments getting outdated due to an automated refactoring.